### PR TITLE
MS-137: Postulants - Implement redux

### PR DESCRIPTION
**Pull request by sofpag**
This resource has some errors that will be fixed in another PR:
-	When you use ADD, the status will be “pending interview” although you write otherwise 
-	When you use EDIT, the date don’t show as it should

### CODE REVIEW

File: src/redux/Postulants/thunks.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  return fetch(BASE_URL, {
+    method: 'POST',
+    mode: 'cors',
+    headers: {
+      'Content-type': 'application/json'
+    },
+    body: JSON.stringify(postulant)
+  })
+    .then((response) => {
+      if (response.status === 200 || response.status === 201) return response.json();
```
Comment (julianv97) [inline src/redux/Postulants/thunks.js:61]: you forgot to throw an Error here

File: src/redux/Postulants/thunks.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    });
+};
+
+const getOnePostulantsFetching = () => ({ type: GET_ONE_POSTULANTS_FETCHING });
+
+const getOnePostulantsFulfilled = (payload) => ({ type: GET_ONE_POSTULANTS_FULFILLED, payload });
+
+const getOnePostulantsRejected = () => ({ type: GET_ONE_POSTULANTS_REJETED });
+
+//FORMA DE JULI
```
Comment (julianv97) [inline src/redux/Postulants/thunks.js:129]: delete this comment

**Review by julianv97 — CHANGES_REQUESTED** [review]

File: src/redux/Postulants/reducer.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        messageText: 'Deleted Postulant',
+        list: state.list.filter((postulant) => postulant.id !== action.payload)
+      };
+    case DELETE_POSTULANTS_REJETED:
+      return {
+        ...state,
+        isLoading: false,
+        error: true,
+        messageType: 'error',
+        messageText: 'Cannot delete Clients'
```
Comment (AndresCespedes23) [inline src/redux/Postulants/reducer.js:95]: Shouldn't be **Postulants** the content of the msg?

Comment (sofpag) [reply src/redux/Postulants/reducer.js:95]: yes!

**Review by AndresCespedes23 — APPROVED** [review]

**Review by julianv97 — APPROVED** [review]

**Pull request merged**